template/src/includes/cpt/example.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Matise\CPT;
+
+class YOUR_CPT_NAME {
```

Comment:
Dit lijkt me vrij onnodig (de cpt vooral) kan je uitleggen waarom je dit hebt gemaakt? Lijkt me erg verwarrend omdat er bijvoorbeeld ook soms case tussenstaat

Reply:
Wat bedoel je met "dit" @Sidstumple 
deze file is toch gewoon een voorbeeld hoe je een Custom Post type kan registreren?

Reply:
Cases is denk ik gewoon gekopieerde zooi die weg mag? @dipsaus9

Reply:
Ja haha bedoel dus die cpt functie maar eigenlijk de hele file

Reply:
Cases weggehaald, denk wel dat het lekker blijft om een voorbeeld te hebben aangezien deze heel vaak gebruikt wordt. 
+ hij wordt niet geinclude ofzo, het is echt een example